### ISSUE TITLE
Position of the Basemap Widget

### ISSUE THREAD

**Issue by drusexton** [cmv/cmv-app#834]
@tmcgee I need to re-position the Basemap widget, because it is obscured by the search widget on a mobile device as shown in the image below. Is this possible? I was trying to do this in the BasemapGallery.css but am not having much success. Thoughts?

![basemapwidgetadjust](https://user-images.githubusercontent.com/14102180/41978314-949c5652-79ef-11e8-9a84-c050d26bbbb7.jpg)

**Comment by drusexton** [comment]
@tmcgee actually I believe I located it (Unless there is some other preferred method you would like to share. ). You can change the location in the js-viewer-templates-mapOverlay.html file as show below. 

![basemappositionhelp](https://user-images.githubusercontent.com/14102180/41981380-c7c6fa9e-79f6-11e8-974c-286ac125c049.jpg)

**Comment by tmcgee** [comment]
@drusexton It is recommended that you not modify the core files as that makes upgrades in the future more difficult. Speaking of upgrades, it looks like you are working with an older version of CMV. In recent versions, there are a number of enhancements to address use via mobile devices:

1. The positioning of on-map widgets like the basemap widget is handled by CSS in the [`cmv.css`](https://github.com/cmv/cmv-app/blob/develop/viewer/css/cmv.css) file and can be overridden within your own custom css.

2. CSS is used to reduce the size of the basemap widget when on phones:

![image](https://user-images.githubusercontent.com/200780/41983327-a24e2d28-79e2-11e8-94f8-a5f2edb0405b.png)

3. We've added a mobile sidebar specifically for phones and touch devices:

![image](https://user-images.githubusercontent.com/200780/41983301-90a65df2-79e2-11e8-848b-9acf1928ff72.png)

4. You can conditionally move widgets to the sidebar for mobile devices. The [`viewer.js`](https://github.com/cmv/cmv-app/blob/develop/viewer/js/config/viewer.js#L353-L368) file for the demo demonstrates this for the Esri Search widget

5. Finally, there are some additional improvements in the works right now specific to the mobile experience:

![image](https://user-images.githubusercontent.com/200780/41983277-73c0b188-79e2-11e8-8cb9-6e9f463cd5eb.png)

**Comment by drusexton** [comment]
@tmcgee Thanks for pointing this out to me. I actually noticed shortly after my previous comments. I went to go "update" the config files for some of my other applications and noticed they have the sidebar config files, and really no need for adjusting the placement of the basemap widget. This particular application was much older then some of the others. Overall it is probably time I download the new or latest config files and update my apps. CHEERS!
